<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseGenGraphForm</class>
 <widget class="QDialog" name="BaseGenGraphForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sofa Graph Export</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="filenameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Output DOT &amp;File</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>filename</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="filename">
       <property name="text">
        <string>untitled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="layoutGroup">
       <property name="title">
        <string>Layout</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QRadioButton" name="layoutDirV">
          <property name="text">
           <string>Directed Vertically</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="layoutDirH">
          <property name="text">
           <string>Directed Horizontally</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="layoutSpring">
          <property name="text">
           <string>Undirected Spring</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="layoutRadial">
          <property name="text">
           <string>Undirected Radial</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="outputsBox">
       <property name="title">
        <string>Outputs</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="genPNG">
          <property name="text">
           <string>Generate PNG</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="genPS">
          <property name="text">
           <string>Generate PS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="genFIG">
          <property name="text">
           <string>Generate FIG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="genSVG">
          <property name="text">
           <string>Generate SVG</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="presetLabel">
       <property name="text">
        <string>Presets</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="presetFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
       <property name="autoCompletion">
        <bool>true</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Full Graph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All Objects</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All Nodes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mechanical Graph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mechanical Objects</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Visual Graph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Visual Objects</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Collision Graph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Collision Objects</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Collision Response Graph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Collision Response Objects</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="filterBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QCheckBox" name="showSolvers">
        <property name="text">
         <string>Solvers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="showMechanicalStates">
        <property name="text">
         <string>Mechanical Models</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="showForceFields">
        <property name="text">
         <string>Force Fields</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="showInteractionForceFields">
        <property name="text">
         <string>Interaction Force Fields</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="showConstraints">
        <property name="text">
         <string>Constraints</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="showMass">
        <property name="text">
         <string>Mass</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="showMechanicalMappings">
        <property name="text">
         <string>Mechanical Mappings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="showTopology">
        <property name="text">
         <string>Topology</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="showMappings">
        <property name="text">
         <string>Mappings</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="showCollisionModels">
        <property name="text">
         <string>Collision Models</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="showVisualModels">
        <property name="text">
         <string>Visual Models</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="showCollisionPipeline">
        <property name="text">
         <string>Collision Pipeline</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="showBehaviorModels">
        <property name="text">
         <string>Behavior Models</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="showContext">
        <property name="text">
         <string>Context</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showObjects">
        <property name="text">
         <string>Objects</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="showNodes">
        <property name="text">
         <string>Nodes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="groupNodeLabel">
       <property name="title">
        <string>Nodes Labels</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="labelNodeName">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="labelNodeClass">
          <property name="text">
           <string>Class</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupObjectLabel">
       <property name="title">
        <string>Objects Labels</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="labelObjectName">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="labelObjectClass">
          <property name="text">
           <string>Class</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="displayButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Display</string>
       </property>
       <property name="shortcut">
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>&amp;Export</string>
       </property>
       <property name="shortcut">
        <string>Alt+E</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>filename</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>layoutDirV</tabstop>
  <tabstop>layoutDirH</tabstop>
  <tabstop>layoutSpring</tabstop>
  <tabstop>layoutRadial</tabstop>
  <tabstop>genPNG</tabstop>
  <tabstop>genPS</tabstop>
  <tabstop>genFIG</tabstop>
  <tabstop>genSVG</tabstop>
  <tabstop>showSolvers</tabstop>
  <tabstop>showMechanicalStates</tabstop>
  <tabstop>showForceFields</tabstop>
  <tabstop>showInteractionForceFields</tabstop>
  <tabstop>showConstraints</tabstop>
  <tabstop>showMass</tabstop>
  <tabstop>showMechanicalMappings</tabstop>
  <tabstop>showTopology</tabstop>
  <tabstop>showMappings</tabstop>
  <tabstop>showCollisionModels</tabstop>
  <tabstop>showVisualModels</tabstop>
  <tabstop>showCollisionPipeline</tabstop>
  <tabstop>showBehaviorModels</tabstop>
  <tabstop>showContext</tabstop>
  <tabstop>showObjects</tabstop>
  <tabstop>showNodes</tabstop>
  <tabstop>labelNodeName</tabstop>
  <tabstop>labelNodeClass</tabstop>
  <tabstop>labelObjectName</tabstop>
  <tabstop>labelObjectClass</tabstop>
  <tabstop>displayButton</tabstop>
  <tabstop>exportButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showBehaviorModels</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showCollisionModels</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showVisualModels</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showMappings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showContext</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showCollisionPipeline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showSolvers</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showMechanicalStates</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showForceFields</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showInteractionForceFields</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showConstraints</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showMass</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showTopology</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>showMechanicalMappings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupObjectLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showNodes</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupNodeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showNodes</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showObjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showBehaviorModels</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showCollisionModels</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showVisualModels</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showMappings</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showContext</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showCollisionPipeline</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showSolvers</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showMechanicalStates</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showForceFields</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showInteractionForceFields</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showConstraints</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showMass</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTopology</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showMechanicalMappings</sender>
   <signal>toggled(bool)</signal>
   <receiver>presetFilter</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
